tests.rb

```diff
@@ -0,0 +1,61 @@
+require 'rspec/autorun'
+require_relative 'book'
+require_relative 'person'
+require_relative 'classroom'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
+
+describe Book do
+  book = Book.new('Abdollah', 'Maramagambo')
+  it 'should return author name and book title' do
+    expect(book.author).to eql('Abdollah')
+    expect(book.title).to eql('Maramagambo')
+  end
+end
+
+describe Person do
+  person = Person.new(1, 18, 'John', parent_permission: true)
+  it 'should return age and name' do
+    expect(person.age).to eql(18)
+    expect(person.name).to eql('John')
+  end
+end
+
+describe Student do
+  student = Student.new(1, 18, 'John', 'Math', true)
+  it 'should return age, name, classroom and parent permission' do
+    expect(student.age).to eql(18)
+    expect(student.name).to eql('John')
+    expect(student.classroom).to eql('Math')
+    expect(student.parent_permission).to eql(true)
+  end
+end
+
+describe Teacher do
+  teacher = Teacher.new(1, 18, 'John', 'Math')
+  it 'should return age, name, classroom and parent permission' do
+    expect(teacher.age).to eql(18)
+    expect(teacher.name).to eql('John')
+    expect(teacher.specialization).to eql('Math')
+  end
+end
+
+describe Classroom do
+  classroom = Classroom.new('Math')
+  it 'has a label' do
+    expect(classroom.label).to eq('Math')
+  end
+end
+
+describe Rental do
+  it 'Test Rentals' do
+    teacher = Teacher.new(1, 18, 'John', 'Math')
+    book = Book.new('Abdollah', 'Maramagambo')
+    rental = Rental.new('2022/10/12', book, teacher)
+
+    expect(rental.date).to eq '2022/10/12'
+    expect(rental.person.name).to eq 'John'
+    expect(rental.book.title).to eq 'Maramagambo'
+  end
+end
```

Comment:
Great job having your tests in the tests.rb file. However, according to the project requirements, you are required touse rspec for the unit tests. This will generally require you to have a spec folder to house all your test cases and import them from there. Below are my requested changes.

- [ ] Create a spec folder and create a corresponding yourfile_spec.rb file eg for the person, it would be person_spec.rb file. Basically break down your test cases into a file corresponding to the class it is supposed to be testing. This way, you will have met all the project requirements. I really this should be your filestructure.

![image](https://user-images.githubusercontent.com/86318284/195406928-c49e199f-1022-4d5b-a5df-6bacfd3b3dee.png)

Reply:
ok, will do it right now